src/main/java/com/theironyard/war/Game.java

```diff
@@ -0,0 +1,126 @@
+        while (warDeck.getSize() > 0){//for (int i = deckSize; i >= 0; i--){
+            //System.out.println(i);
+            player1.takeCard(warDeck.dealCard());
+            player2.takeCard(warDeck.dealCard());
+        }
+
+        System.out.format("%s has %d cards\n", player1.getName(), player1.getHandSize());
+        System.out.format("%s has %d cards\n", player2.getName(), player2.getHandSize());
+
+    }
+
+    /**
+     * Starts each round of the game. Check for a winner a winner of each round and calls declareWinner() & war().
+     */
+    public void startRound(){
+        boolean gameOver = false;
+        setupPlayers();
+
+        int roundCounter = 0;
+        do {
+            roundCounter++;
+            System.out.printf("\n********** Round %d **********\n", roundCounter);
+            System.out.format("%s has %d cards, plays %s\n", player1.getName(), player1.getHandSize(), player1.getCardInfo(TOPOFDECK));
+            System.out.format("%s has %d cards, plays %s\n", player2.getName(), player2.getHandSize(), player2.getCardInfo(TOPOFDECK));
+
+            if (player1.getCardValue(TOPOFDECK)> player2.getCardValue(TOPOFDECK)) {
+                System.out.format("%s wins with %s\n", player1.getName(), player1.getCardInfo(TOPOFDECK));
+                player1.takeCard(player2.giveCard());
+                gameOver = (player2.getHandSize() == 0);
+            } else if (player2.getCardValue(TOPOFDECK) > player1.getCardValue(TOPOFDECK)) {
+                System.out.format("%s wins with %s\n", player2.getName(), player2.getCardInfo(TOPOFDECK));
+                player2.takeCard(player1.giveCard());
+                gameOver = (player1.getHandSize() == 0);
+            }else if (allowWar()){
+                war(player1, player2);
+            }
+            else
+                gameOver = true;
+        }while(gameOver == false);
+        declareWinner();
+    }
+
+    /**
+     * Runs the war part of the game. If chooses a winner and gives them the cards in the pot.
+     * @param player1 - first player
+     * @param player2 - second player
+     */
+    public void war(Player player1, Player player2){
+        List<Card> winnersPot = new ArrayList<Card>();
+        System.out.println("\n********** WAR!!! Get SOME! **********");
+        for (int i = 0; i < 4; i++) {
+            winnersPot.add(player1.giveCard());
+            winnersPot.add(player2.giveCard());
+        }
+
+        System.out.format("%s has %d cards, plays %s\n", player1.getName(), player1.getHandSize(), player1.getCardInfo(TOPOFDECK));
+        System.out.format("%s has %d cards, plays %s\n", player2.getName(), player2.getHandSize(), player2.getCardInfo(TOPOFDECK));
+
+        if (player1.getCardValue(TOPOFDECK)> player2.getCardValue(TOPOFDECK)){
+            System.out.format("%s wins with %s\n", player1.getName(), player1.getCardInfo(TOPOFDECK));
+            for (Card card : winnersPot) {
```

Comment:
This block should point you in the direction of creating a `takeCard` method that takes multiple cards.  Especially, since there is an `addAll` method on lists that takes a collection and adds all elements.